jolly-jellyfish/src/page_maker/templates/page_maker/template_list.html

```diff
@@ -0,0 +1,11 @@
+{% extends "page_maker/base.html" %}
+
+{% block content %}
+{% for template in templates %}
```

Comment:
```suggestion
{% block content %}
<h2>Templates</h2>
{% for template in templates %}
```
As above, styling concerns.